### PR TITLE
feat(ERF-58): POI tooltip

### PR DESCRIPTION
**Pull request by dnsos**
This PR adds the tooltip for the POIs.

### CODE REVIEW

File: src/components/MapPoiTooltip/index.tsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  coordinates,
+}) => {
+  return (
+    <Popup
+      latitude={coordinates.latitude}
+      longitude={coordinates.longitude}
+      closeButton={false}
+      className="w-64 rounded-2xl"
+    >
+      {title && <h4 className="text-xl leading-6 text-gray-900">{title}</h4>}
```
Comment (vogelino) [inline src/components/MapPoiTooltip/index.tsx:27]: These conditions here imply that there is the possibility that a tooltip without content is shown. I would add a condition that prevents the tooltip to show at all or that indicates that no data is available if neither `title`, `category`, nor `info` are available.

File: src/components/MapPoiTooltip/index.tsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  category,
+  info,
+  coordinates,
+}) => {
+  return (
+    <Popup
+      latitude={coordinates.latitude}
+      longitude={coordinates.longitude}
+      closeButton={false}
+      className="w-64 rounded-2xl"
```
Comment (vogelino) [inline src/components/MapPoiTooltip/index.tsx:25]: Feel free to ignore, but I'm not convinced about the aesthetic of the hugely rounded corner. It is not really matching the rest of the UI. Also, I think increasing the tooltip's padding should make its content breathe a little better.

File: src/modules/RefreshmentMap/index.tsx [hunk 3]
```diff
@@ -34,6 +39,51 @@ export const RefreshmentMap: FC<RefreshmentMapPropType> = (pageProps) => {
...
+      source: string
+      properties: {
+        name?: string
+        category?: string
+        info?: string
+      }
+      [key: string]: unknown
+    }
+    interface CustomMapEventType extends MapEvent {
+      features: MapFeature[]
```
Comment (vogelino) [inline src/modules/RefreshmentMap/index.tsx:63]: I'm not sure why these interfaces are defined within the function here. You might want to move those to the top of the file to help the reader focus on the functionality. Or is there another reason?

**Review by vogelino — CHANGES_REQUESTED** [review]
Great work! It definitively feels great to know what all these POIs are now! 😆 I added a few comments for the sake of annoying you a little bit 😝 Feel free to ignore them if you want.

File: src/modules/RefreshmentMap/index.tsx (hunk 3 above)
Comment (dnsos) [reply src/modules/RefreshmentMap/index.tsx]: There *was* a reason while working on this. The `features` in `MapEvent` were typed as `any[]`, so it took me quite some trial and error to understand how to make that typing more expressive for us. During that process it was easier to have the types closeby. 

I changed it now.

File: src/components/MapPoiTooltip/index.tsx (hunk 2 above)
Comment (dnsos) [reply src/components/MapPoiTooltip/index.tsx]: Good points. I adjusted that.

File: src/components/MapPoiTooltip/index.tsx (hunk 1 above)
Comment (dnsos) [reply src/components/MapPoiTooltip/index.tsx]: True, these conditions actually don't make sense. I've made a change now that requires `title` and `category` in `MapPoiTooltip` as they are crucial for understanding the POI. Without these provided, the component won't be rendered at all.

**Review by vogelino — APPROVED** [review]
Great!

**Pull request merged**